pymilo/utils/util.py

```diff
@@ -119,3 +119,19 @@ def has_named_parameter(func, param_name):
     _signature = signature(func)
     parameter_names = [p.name for p in _signature.parameters.values()]
     return param_name in parameter_names
+
+
+def prefix_list(a, b):
+    """
+    Check whether the b list is a sublist of the a list.
+
+    :param a: outer list
```

Comment:
I recommend using `list1` and `list2` instead of `a` and `b`.

Reply:
fixed in [a600f62](https://github.com/openscilab/pymilo/pull/104/commits/a600f6272c3a50422d6ef57f75d124ba6c63694a).